src/index.ts

```diff
@@ -121,16 +119,20 @@ export interface TableMetadata {
   };
 }
 
-export interface FileUploadOptionsSpec {
+export interface TableUploadOptionsSpec {
   type: UploadType;
```

Comment:
This is always `csv` with tables, so I think this can go too
```suggestion
```

Reply:
Got it!
See a956e4a